README.md

```diff
@@ -86,8 +101,38 @@ mapping (address => uint256) public liquidity;
 
 </details>
 
-These variables track the total liquidity, but also by individual addresses too.
-Now, let's create an `init()` function in `DEX.sol` that is payable and then we can define an amount of tokens that it will transfer to itself.
+</details>
+
+These variables track the total liquidity, but also the liquidity of each address.
+Now, let's create an `init()` function in `DEX.sol`. 
+We want this function written in a way that when we send ETH and/or $BAL tokens through our front end or deployer script, the function will get those values from the contract and assign them onto the global variables we just defined. 
+
+<details markdown='1'><summary>🦉 Guiding Questions</summary>
+
+<details markdown='1'><summary>Question 1</summary>
```

Comment:
Question 1-4, but Question One, Two etc in other places

Reply:
fixed your 2 other comments, what is the problem with this one? i did not understand

Reply:
Sorry, didn't see your answer. Just wording. In one place questions written `Question 1`, `Question 2`, ... (see above). In all other places  `Question One`, `Question Two`, ...